snap/local/README.md

```diff
@@ -7,9 +7,9 @@
 
```

Comment:
Why is the README and build.sh moved into `snap/local/`? Should they be deleted instead..?

Reply:
Here is error output from snapcraft:

```
The 'snap' directory is meant specifically for snapcraft, but it contains the following non-snapcraft-related paths, which is unsupported and will cause unexpected behavior:
- build.sh

If you must store these files within the 'snap' directory, move them to 'snap/local', which is ignored by snapcraft.
```

Reply:
Then please delete the files. I'll merge this PR after you've done that.